### PR TITLE
Spiderplush

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
Literally staying up past 2:00 AM to code in spider plushie.

Has been included into arcade prizepools and other toy-inclusive loot tables.
Also triggers arachnophobia as it should.

 
Credit to whatever spriter made the item sprite, hit me up and I'll give proper cred.
Inhands sprites by me.

## Why It's Good For The Game
4 times the hugs

## Changelog
:cl:
add: Added a cute spider plush.
/:cl:

### CODE REVIEW

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by AsciiSquid** [comment]
**UPDATE:** Spider plushies are now loaded with silly-string webbing.
image
It doesn't slow you down but it is another thing for janitors to clean up.

The plush can be fed plastic sheets to refill the spray when it runs out.

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -698,3 +698,31 @@
...
+		var/usedcharges = min(max_spraycharges - spraycharges, p_sheets.get_amount())
+		if(p_sheets.use(usedcharges))
+			spraycharges += usedcharges
+			playsound(src.loc, 'sound/weapons/bite.ogg', 40, TRUE, -6)
+			to_chat(user, "<span class='notice'>You feed [src] [usedcharges] sheets of plastic.</span>")
+
+/obj/item/toy/plush/spider/afterattack(atom/A, mob/user)
+	. = ..()
+	if(istype(A, /turf/open/floor))
+		if(!spraycharges)
```
Comment (MarkSuckerberg) [inline code/game/objects/items/plushes.dm]:
```suggestion
		if(spraycharges <= 0)
```
I think it's better to be safe than sorry

**Comment by AsciiSquid** [comment]
Integration failure is in piping components, something spider plushcode doesn't touch.

**Comment by TerryTehCoder** [comment]
47 lines of code and 8 files for a silly string spider plushie. The Future Really IS now in Dream Maker.

**Comment by ZephyrTFA** [comment]
Can you push an empty commit to try and get CI to be happy

**Comment by ZephyrTFA** [comment]
Haha, CI is willing today

**Pull request merged**